vagrant/.gitignore

```diff
@@ -1 +1,3 @@
-.vagrant
\ No newline at end of file
+.vagrant
+*.box
+Berksfile.lock
```

Comment:
i'm not a chef maven, but i'm pretty sure we're supposed to commit the .lock file too -- @eherot or @jeffbyrnes might know best